This patch removes the <toolset> decoration from all library names.

--- upstream-boost-1.35.0/Jamroot	2008-03-25 15:47:24.000000000 -0500
+++ boost-1.35.0/Jamroot	2008-04-13 19:19:07.000000000 -0500
@@ -309,8 +309,8 @@
         if $(layout) = versioned
         {
             local result = [ common.format-name
-                <base> <toolset> <threading> <runtime> -$(BOOST_VERSION_TAG)
+                <base> <threading> <runtime> -$(BOOST_VERSION_TAG)
                 -$(BUILD_ID)
                 : $(name) : $(type) : $(property-set) ] ;
             
